README.md

```diff
@@ -5043,6 +5043,10 @@ Creates a VXLAN Network Virtualization Endpoint (NVE) overlay interface that ter
 |---------------------------------|--------------------------------------|
 | source_interface_hold_down_time | Not supported on N3k, N5k, N6k <br> Supported in OS Version 8.1.1 and later on N7k |
 | multisite_border_gateway_interface | Only supported on N9K-EX and N9K-FX devices. For eg: N9K-C93180YC-EX. Minimum OS version 7.0(3)I7(1) and minimum Module Version 1.9.0 |
+| global_suppress_arp             | Only supported on N9K and N9K-F running OS Version 9.2 and later |

```

Comment:
Here and elsewhere.  This should be `9.1 and later` right?

Also, I would not mention `N9k-F` in the documentation for any of these new properties.  There is really no distinction as far as this is concerned.  They are all N9k's running 9.1 or later.

Reply:
As mentioned in the NU, the CCO images are only 9.2.1 and 9.2.2, no 9.1

Reply:
About N9k vs N9k-F, there MUSt be a distinction some properties are supported on N9k and some on both N9k and N9k-F. If I remove it, then how do we say that some are not supported on N9k-F? We can discuss this if you are not clear